napalm_iosxr/constants.py

```diff
@@ -0,0 +1,23 @@
+"""Constants for the IOS-XR driver."""
+
+from __future__ import unicode_literals
+
+from napalm_base.constants import *  # noqa
+
+SR_638170159_SOLVED = False
```

Comment:
I'll change this when the bug will be solved (although I assume the answer is: "never"). But let's not underestimate Cisco!

I introduced this flag (https://github.com/napalm-automation/napalm-iosxr/pull/82/commits/5954c2f361d5914ab0b84f163a9c7b1f5dcd1743) just because I didn't want to remove the entire part of code that comes after L#1281.